terraform-modules/digitalocean/load_balancer/README.md

```diff
@@ -0,0 +1,2 @@
+# Module under construction 
+🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃🏃
```

Comment:
Do we need this?  Wouldnt the digital ocean's kubernetes provider automagically create the LB for you?